pkg/virt-handler/node-labeller/BUILD.bazel

```diff
@@ -45,6 +45,7 @@ go_test(
     ],
     data = glob(["testdata/**"]),
     embed = [":go_default_library"],
+    race = "on",
```

Comment:
What about other tests?

Reply:
Yeah I was surprised by the change in behaviour, assuming this does actually drop coverage of the non-cgo components I'll document this clearly in the commit.